src/components/events/EventCard.old.tsx

```
@@ -0,0 +1,426 @@
+  View, 
+  Text, 
+  TouchableOpacity, 
+  StyleSheet, 
+  Image,
+  ActivityIndicator,
+  Alert
+} from "react-native";
+import { useNavigation } from "@react-navigation/native";
+import { format } from "date-fns";
+import { Ionicons } from "@expo/vector-icons";
+import { Event, eventService } from "../../lib/eventService";
```

Biome (1.9.4)

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)